### PR TITLE
Add the Ameridian Tender

### PR DESCRIPTION
**Pull request by r4d6**
## About The Pull Request
Add the Ameridian Tender, a small, ameridian-powered automaton, who's sole purpose in life is to harvest fully-grown ameridian crystals that have spread a bit. Due to its nature, it doesn't get attacked by ameridian golems and isn't damaged by sonic weaponry. 

Ghosts can take control of non-possessed tenders by using the 'Respawn as ameridian tender' verb in the ghost panel.

Misc Changes : 
\- Reduce cost & price of ameridian cores, and buff the amount of research they give.
\- Ameridian crystals now take into account the tool quality level.
\- Rename Ion Jaunter to Ion Jaunt.
\- Add a whitelist check to the mice respawn verb

### CODE REVIEW

**Comment by IrkallaEpsilon** [comment]
That a tiberium harvester?

**Comment by r4d6** [comment]
> That a tiberium harvester?

A smol & cute harvester, yes.

File: code/modules/ameridian/ameridian_crystal.dm [hunk 1]
```diff
@@ -58,14 +58,10 @@
...
-		src.visible_message(SPAN_NOTICE("[user] starts excavating crystals from [src]."), SPAN_NOTICE("You start excavating crystal from [src]."))
-		if(do_after(user, WORKTIME_NORMAL, src))
-			var/obj/item/stack/material/ameridian/T = new(get_turf(src))
-			T.amount = growth // Drop more crystal the further along we are
-			src.visible_message(SPAN_NOTICE("[user] excavates a crystal from [src]."), SPAN_NOTICE("You excavate a crystal from [src]."))
-			activate_mobs_in_range(src, 15) // Wake up the nearby golems
-			qdel(src)
+	if(user.a_intent == I_HELP && user.Adjacent(src))
+		visible_message(SPAN_NOTICE("[user] starts digging [src] up."), SPAN_NOTICE("You start digging [src] up."))
+		if(I.use_tool(user, src, WORKTIME_NORMAL, I.get_tool_type(user, list(QUALITY_EXCAVATION, QUALITY_DIGGING, QUALITY_SHOVELING), src), FAILCHANCE_EASY))
```
Comment (lolman360) [inline code/modules/ameridian/ameridian_crystal.dm:63]:
```suggestion
		if(I.use_tool(user, src, WORKTIME_NORMAL, I.get_tool_type(user, list(QUALITY_EXCAVATION, QUALITY_DIGGING, QUALITY_SHOVELING), src), FAILCHANCE_EASY, required_stat = STAT_ROB))
```

**Comment by r4d6** [comment]
Ah fuck it, there's no way to get it In-Game currently, so it's fine if the sprite isn't final.

**Pull request merged**